src/javascript/app/base/page.js

```diff
@@ -133,13 +134,16 @@ const Page = (() => {
                 }, 1000);
                 RealityCheck.onLoad();
                 Menu.init();
+                const is_at_brazil = State.getResponse('website_status.clients_country') === 'br';
```

Comment:
shouldn't it be like this:
`                const is_at_brazil = (Client.get('residence') === 'br' ||State.getResponse('website_status.clients_country') === 'br');
`
also better to name it `is_br_client`

Reply:
I just wanted client which they access their account in brazil.